examples/grad/lv_example_grad.h

```diff
@@ -1,10 +1,10 @@
 /**
- * @file lv_draw_sw_gradient_private.h
+ * @file lv_example_grad.h
  *
  */
 
-#ifndef LV_DRAW_SW_GRADIENT_PRIVATE_H
-#define LV_DRAW_SW_GRADIENT_PRIVATE_H
+#ifndef LV_EX_GRAD_H
+#define LV_EX_GRAD_H
```

Comment:
```suggestion
#ifndef LV_EXAMPLE_GRAD_H
#define LV_EXAMPLE_GRAD_H
```

(and at the bottom of the file)